src/TwigComponent/doc/index.rst

```diff
@@ -840,6 +840,117 @@ The ``with`` data is what's mounted on the component object.
 .. note::
 
     Embedded components *cannot* currently be used with LiveComponents.
+    
+Component HTML Syntax
+-----------------
+    
+.. versionadded:: 2.9
+
+    This syntax was been introduced in 2.8 and is still experimental: it may change in the future.
+    
+Twig Component came with a new syntax to ease the readability of your template:
+
+.. code-block:: html+twig
+
+    {# templates/some_page.html.twig #}
+    <twig:Alert warning :user="user.username" message="I am an alert!">
+       <div class="alert-content">
+            // ... your component content
+       </div>
+       <twig:block name="footer">
+            // ... footer content
+       </twig:block>
+    </twig:Alert>
+    
+You can now use a custom tag prefix starting by <twig: and your component name
```

Comment:
I would remove this sentence and the example above. 

So it would be:

> Twig Components come with an HTML-like...

And then the simple example below